### ISSUE TITLE
[General] Steward for Marketing Documentation

### ISSUE THREAD

**Issue by caitmm43** [microprofile/microprofile-marketing#632]
Opening a new ticket to preserve this info, as the previous ticket #601 is now closed.  

@aeiras to @nsardana-bny on Dec. 8th: 
> --- 1 thing I would like to submit as a call to action for Neha and Neha you can say, "no bandwidth" for 2021 or similar.
>
>@nsardana-bny, Neha,
>It could be great if you choose to become the Steward on the Marketing documentation. There is no need to wait for a call. Either we use a new ticket or the mailing list to discuss.
>
>I wrote the MPWG Onboarding members currently link to the MPWG page https://microprofile.io/workinggroup/
https://github.com/microprofile/microprofile-wg/wiki/MicroProfile-Working-Group:--Members-Onboarding
>
>Ideally, other members help with the parts within.
For example, Committer Rep Onboarding wiki thread https://groups.google.com/g/microprofile-marketing/c/YePKMccVhhc
>
>The Marketing wiki follows the same form. It ought to be inserted into the main Onboarding document. All the specific wiki's will make the document cleaner, shorter and much more consumable at one try.
>
>Now, the marketing draft (gathers all the current stuff (documented) under Marketing into 1 document) ought to be drafted under Community documentation and reviewed before being exported as a standalone wiki under MPWG repo and linked to the Main document of onboarding new members.
>
>openly brainstorming. Thank you for facilitating the exchange, amazing!

**Comment by nehasardana09** [comment]
I have started working on the wiki page for Mktg-Call Steward Onboarding & Call Steward Responsibilities. I have access to edit and save the wiki page. Will close this ticket once it is complete. 
The document I am using to convert is https://docs.google.com/document/d/1ayqaFoXGeTZdneL6U9oekaZP3LPe7PapTlg7jh7eD3M/edit#heading=h.bgmy0hc461ot
and the location on github is https://github.com/microprofile/microprofile-wg/wiki/Mktg-Call-Steward-Onboarding-&-Call-Steward-Responsibilities---Work-in-progress

Please dont close the ticket yet. I am still working on it.

**Comment by caitmm43** [comment]
@nsardana-bny As I am going through onboarding now, I would love to share some of the questions and answers I found.  Do you have a specific timeline in mind to complete this?

**Comment by nehasardana09** [comment]
Sure @caitmm43. Please feel free to edit the word document. We can incorporate those in the Wiki.

**Comment by caitmm43** [comment]
@nsardana-bny will do!  I've requested edit access to that doc.  Will update with suggestions after the break.

**Comment by caitmm43** [comment]
@nehasardana09 A note for later:  The current template for minutes is different from the desired format that you and Debbie reworked.  I noticed this after I used the old template, and Amelia pointed out some edits needed.  We should address this in the new onboarding wiki and merge/update the template linked in the onboarding responsibilities doc.

**Comment by caitmm43** [comment]
Note: It was decided on the Feb 1st MP Live Hangout call that closed tickets from the marketing minutes should no longer be pasted into to the MP Live Hangout Minutes.

**Comment by aeiras** [comment]
direct feedback that we apply on the spot. :) Thank you Neha & Caitlin for working on the wiki, you two are amazing!

**Comment by caitmm43** [comment]
Note for later: update or replace the minutes template currently in drive.  Has some old formatting (for example: 4 columns including one for category vs. desired 3 columns without to free up space).

**Comment by aeiras** [comment]
Hey beautiful @nsardana-bny & @caitmm43, 

The wiki exists https://github.com/microprofile/microprofile-wg/wiki/Mktg-Call-Steward-Onboarding-&-Call-Steward-Responsibilities---Work-in-progress

Thank you for choosing to document lots of the process. :)
